src/screens/ExchangeModal.js

```diff
@@ -32,8 +32,10 @@ import {
 import { FloatingPanel } from '../components/floating-panels';
 import { GasSpeedButton } from '../components/gas';
 import { Centered, KeyboardFixedOpenLayout } from '../components/layout';
-import { FLASHBOTS } from '../config/experimental';
-import useExperimentalFlag from '../config/experimentalHooks';
+/*
+  import { FLASHBOTS } from '../config/experimental';
```

Comment:
keep this cause its tbd what we're gonna be doing regarding flashbots

Reply:
localstorage is fine so I'm ok to remove it